code/index.html

```diff
@@ -2,21 +2,163 @@
 <html>
   <head>
     <meta charset="utf-8" />
-    <title>Project 2: The News Site</title>
+    <title>Arctic News by Ekaterina Klimenko</title>
     <link rel="stylesheet" href="./style.css" />
-    <!-- dont forget the meta tag for responsiveness -->
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
   </head>
   <body>
-    <header>
-      <!-- Some kind of logo and navbar -->
+    
+    <header class="header-general">
+      <!-- Header phone: hamburger menu and dropdown navbar -->
+      <div class="header-phone">
+          <img src="./Arctic-Logo.png" class="logo" alt="Arctic News">
+          <div class="checkbox">
+            <input type="checkbox" id="hamburger">
+              <nav class="dropdown">
+                <ul class="dropdown-list">
+                    <li><a href="" class="nav-link">Home</a></li>
+                    <li><a href="" class="nav-link">Highlights</a></li>
+                    <li><a href="" class="nav-link">News</a></li>
+                    <li><a href="" class="nav-link">More</a></li>
+                </ul>
+              </nav>
+            <label class="label" for="hamburger">
+              <div class="hamburger-line"></div>
+              <div class="hamburger-line"></div>
+              <div class="hamburger-line"></div>
+            </label>
+          </div>
+      </div>
+      <!-- Header tablet: logo in the middle, navbar on the sides-->
+      <div class="header-tablet">
+        <nav class="nav-tablet">
+          <a href="" class="nav-link">Home</a>
+          <a href="" class="nav-link">Highlights</a>
+          <img src="./Arctic-Logo.png" class="logo" alt="Logo">
```

Comment:
The `nav` should contain only navigation elements, so the logo can be outside

Reply:
I have a question. If an image is wrapped inside an <a> tag and technically a link to home page could it be part of nav? I redid my nav now without an image, but was just curious whether this is something that could be done differently in the future?

Reply:
That's a good question! I guess technically, it's part of the navigation then so I would say that it's OK